io.github.cboxdoerfer.FSearch.yml

```diff
@@ -0,0 +1,38 @@
+app-id: io.github.cboxdoerfer.FSearch
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: fsearch
+build-options:
+    build_args:
+        - --share=network
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  # Wayland access
+  - --socket=wayland
+  # Needs to save files locally
+  - --filesystem=host
```

Comment:
this is a very broad permission.

Reply:
I've thought about this, but this is a file search utility and it wouldn't be of much use if it can't find files outside of the users home directory.